src/monodroid/jni/mkbundle-api.h

```diff
@@ -10,7 +10,7 @@ typedef struct BundleMonoAPI
 	void (*mono_register_bundled_assemblies) (const MonoBundledAssembly **assemblies);
 	void (*mono_register_config_for_assembly) (const char* assembly_name, const char* config_xml);
 	void (*mono_jit_set_aot_mode) (int mode);
-	void (*mono_aot_register_module) (void* aot_info);
+	void (*mono_aot_register_module) (void** aot_info);
```

Comment:
Er...what?  Did we always have the wrong declaration?  Did Mono change the declaration?

Reply:
It looks like we had it wrong from the beginning. The signature of the function in Mono is:

```C
void mono_aot_register_module (gpointer *aot_info);
```

It was added in [this commit](https://github.com/mono/mono/blob/6c8eefbd482aef9ea51be7ab400bd02f79baa263/mono/mini/aot-runtime.c#L818-L819) to the `C` code and in [this commit](https://github.com/mono/mono/blob/0e0f1b9be08ff14b192fdadc66efb6bb763867a4/mono/mini/jit.h#L106-L107) to `jit.h` using `void*` instead of `gpointer`.